src/app/components/authorized/add-new-subreddit/add-new-subreddit.component.spec.ts

```diff
@@ -1,31 +1,55 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { HttpClient } from '@angular/common/http';
 import { FormsModule } from '@angular/forms';
-import { Router } from '@angular/router';
+import { ExtraOptions, Router, Routes } from '@angular/router';
 import { AddNewSubredditComponent } from './add-new-subreddit.component';
 import { GatewayService } from 'src/app/services/gateway.service';
+import { SubredditCreation } from 'src/types/subreddits';
+import { ModuleWithProviders } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { RouterTestingModule } from '@angular/router/testing';
+import { FeedComponent } from '../feed/feed.component';
 
-class MockDataService {}
+class MockService {
```

Comment:
MockService doesnt say much about what is being mocked.

Reply:
changed name as requested